CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 2.15.2
+
+* Change tile hash function to fnv1a
```

Comment:
Is this elsewhere? These commits only seem to change the logging

Reply:
it's already on main: https://github.com/felt/tippecanoe/commit/bfabf8325f96fcf9a2b43170b0472fc58db8f04d

Reply:
Sorry, yes, this is just bringing the changelog up to date with what is already merged